src/main/webapp/index.html

```diff
@@ -26,9 +26,9 @@
         <button id="select-text">Text</button>
         <button id="select-blob">Upload Image</button>
       </div>
-      <form id="keyword-form" class="hide" action="/keyword" method="post" enctype="multipart/form-data">
+      <form id="keyword-form" class="hide" action="/keyword" method="post" enctype="multipart/form-data" onsubmit="validateForm(event);">
```

Comment:
[Nit] Do not combine markup with javascript. Instead of using onsubmit add an event listener.